src/graphic-device/dot-use-examples/index.md

```diff
@@ -190,28 +190,15 @@ Indicative examples for illustrative purposes only.
 
 <!-- Obviously, that's not the right heading, needs proper design -->
 
-<div class="app-inform-inspire">
-<p class="govuk-visually-hidden">From most infomative to most inspiring</p>
-<p class="govuk-heading-s app-inform-inspire__inform" aria-hidden="true">Inform</p>
-
-{% figure { src: "./video-thumb-get-help.png", alt: "A video thumbnail for 'Get help with registering a death'. Title is shown within a simple blue circle graphic on a blue background.", classes: "govuk-!-text-align-left" } %}
+{% informInspire { contentClasses: 'app-inform-inspire__content--flex app-inform-inspire__content--flex-gap-5' } %}
```

Comment:
Looks like the wrong class name is used here? The last one appears to be `.app-inform-inspire__content--flex-5-items` in the CSS file.